addons/api/addon/models/account.js

```diff
@@ -5,20 +5,35 @@
 
 import GeneratedAccountModel from '../generated/models/account';
 import { computed } from '@ember/object';
-import { equal } from '@ember/object/computed';
+import {
+  TYPE_AUTH_METHOD_PASSWORD,
+  TYPE_AUTH_METHOD_OIDC,
+  TYPE_AUTH_METHOD_LDAP,
+} from './auth-method';
```

Comment:
Im currently using the same type values declared in the auth-method model since an account will always have the same type as its auth-method. Is this reasonable or should this account model have its own type values.

Reply:
I think it is reasonable to import this since we will always have the same type as auth-methods